### PR TITLE
mulle: Configure hardware CS for all on-board SPI slaves

### PR DESCRIPTION
**Pull request by gebart**
Follow up on #6572. Define the parameters for SPI0 hardware CS 1,2,3 pins in addition to the original HWCS 0.

### CODE REVIEW

File: boards/mulle/include/periph_conf.h [hunk 1]
```diff
@@ -265,9 +265,9 @@ static const spi_conf_t spi_config[] = {
         .pin_clk  = GPIO_PIN(PORT_D, 1),
         .pin_cs   = {
             GPIO_PIN(PORT_D, 0),
-            GPIO_UNDEF,
-            GPIO_UNDEF,
-            GPIO_UNDEF,
+            GPIO_PIN(PORT_D, 4),
+            GPIO_PIN(PORT_D, 5),
```
Comment (PeterKietzmann) [inline boards/mulle/include/periph_conf.h]: Without going into details, are you sure this is `D5` and not `D0`? Unfortunately I don't have a mulle to test.

Comment (gebart) [inline boards/mulle/include/periph_conf.h]: Yes PTD5 is correct.
PTD0 is SPI0_PCS0, PTD4 is SPI0_PCS1, PTD5 is SPI0_PCS2. PTD6 is SPI0_PCS3

**Comment by gebart** [comment]
@PeterKietzmann rebased to make this PR independent. This is currently blocking Mulle support on #6762.

**Comment by gebart** [comment]
Thanks 🎂

**Pull request merged**